pipeline/README.md

```diff
@@ -4,20 +4,20 @@
 
 ## Bootstrapping
 
-Initial install for a cluster must happen in fybrik-system (currently).  It won't hurt anything if you aren't sure, and reinstall in fybrik-system.
-```
-. source-external.sh
-bash -x bootstrap-pipeline.sh fybrik-system
-# follow on screen instructions
-```
+Run the following command to bootstrap a Fybrik instance. By default the instance of Fybrik will be deployed as namespaced scoped, meaning the controller will only watch and process FybrikApplications from a specific namespace. Therefore limiting you to create FybrikApplications from one namespace. By that namespace is the one specific on the bootstrap script (a.k.a *system* namespace). You can change this behavior by setting environment variables. For details see [Scoping the deployment](#scoping-the-deployment).
 
-Subsequent installs can go in any namespace
+The parameter specified is the *system* namespace. 
 ```
 . source-external.sh
 bash -x bootstrap-pipeline.sh fybrik-myname
 # follow on screen instructions
 ```
 
+After running the "tkn" command that was returned from the bootstrap script, the following namespaces will be created. Illistrated by using the example *system* namespace of fybrik-myname:
+1. fybrik-myname - This will be the *system* namespace where Fybrik is deployed. By default the controller will only process FybrikApplications from this namespace.
+2. fybrik-myname-blueprints - this is the designated namespace for blueprints custom resources. This is also where the data access module is deployed to. 
+3. fybrik-myname-app - This namespace is only created if running namespace scoped AND the following environment variable is set: use_application_namespace=true.  
+
```

Comment:
@laurieaw63 @roee88 What about the namespaces for the data plane for each workload?  Should this be configurable as well?

Reply:
@simanadler What do you mean? This PR does not introduce separate data plane namespace per fybrik application

Reply:
@roee88 is correct this PR is only addressing allowing the name of the blueprint namespace to be configurable; as well as allowing Fybrik to be setup such that it only accepts FybrikApplications from a specific namespace.